src/ophys_etl/transforms/convert_rois.py

```diff
@@ -0,0 +1,159 @@
+from pathlib import Path
+
+from argschema import ArgSchema, ArgSchemaParser
+from argschema.fields import List, InputFile, Str, Dict, OutputFile, Float
+import marshmallow as mm
+import numpy as np
+import h5py
+
+from ophys_etl.transforms.roi_transforms import (binarize_roi_mask,
+                                                 suite2p_rois_to_coo,
+                                                 coo_rois_to_old)
+from ophys_etl.transforms.data_loaders import get_max_correction_border
+
+
+class BinarizeAndCreationException(Exception):
+    pass
+
+
+class BinarizeAndCreateROIsInputSchema(ArgSchema):
+
+    suite2p_stat_path = Str(
```

Comment:
I know someone will ask about why this is Str and not InputFile, argschema will fail validating these files as it uses python `open` on windows to check for ability to read and this does not work with .npy or .h5 files.

Reply:
You can create a custom argparse method that doesn't use open (say, uses os.path.exists) if you want to ensure this file exists.

Reply:
I used a marshmallow post load to validate existence it's just less ideal than having the InputFile

Reply:
Why not define a new marshmallow field class that inherits from and modifies the behavior of the `InputFile` class?

Here is the definition of that class btw: https://github.com/AllenInstitute/argschema/blob/afc2b9241176a587e543f71830f2968d7a552cb7/argschema/fields/files.py#L188

Reply:
Here's a snippet of the stack trace when either `stat.npy` or 'movie.h5` is an InputFile in the schema.
```
C:\Users\mallo\AppData\Local\Programs\Python\Python38\lib\site-packages\argschema\fields\files.py:196: in _validate
    validate_input_path(value)
C:\Users\mallo\AppData\Local\Programs\Python\Python38\lib\site-packages\argschema\fields\files.py:158: in validate_input_path
    s = f.read()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <encodings.cp1252.IncrementalDecoder object at 0x0000023370205B50>
input = b'\x93NUMPY\x01\x00v\x00{\'descr\': \'|O\', \'fortran_order\': False, \'shape\': (10,), }                             ...\x13\x00\x00\x00\x13\x00\x00\x00\x13\x00\x00\x00\x13\x00\x00\x00\x13\x00\x00\x00\x13\x00\x00\x00q\xd6tq\xd7buetq\xd8b.'
final = True

    def decode(self, input, final=False):
>       return codecs.charmap_decode(input,self.errors,decoding_table)[0]
E       UnicodeDecodeError: 'charmap' codec can't decode byte 0x90 in position 311: character maps to <undefined>```

I can actually get this to replicate by just doing the following to those files defined in the `conftest.py` file by running the following:
```python
stat_path, stat_params = s2p_stat_fixture
with open(stat_path) as f:
    f.read()
```

Reply:
I suspect if you used `open(file, mode='rb')` it would work.

Reply:
I don't want the overhead of defining a new marshmallow class and the validation function, it's not hard but it's extra that doesn't need to be added. I'll just stick with the post load and str as input for those two fields as InputFile is doing exactly what I'm doing in post load anyway, more or less.

Reply:
Added an issue to argschema to fix this. Should be simple fix:
https://github.com/AllenInstitute/argschema/issues/105